fetch.bs

```diff
@@ -160,35 +160,40 @@ lt="authentication entry">authentication entries</a> (for HTTP authentication).
 
 <hr>
 
-<p><a>Tasks</a> that are
-<a lt="queue a task">queued</a> by this standard are annotated as one
-of:
+<p>A <dfn>fetch params</dfn> is a <a for=/>struct</a> used in <a for=/>fetching</a>. It has the
```

Comment:
Maybe expand on its use a little bit to explain that it's used internally in this standard? Based purely on the name, one might assume it's something callers of fetch need to know about.